internal/cmdget/cmdget.go

```diff
@@ -42,11 +43,9 @@ func NewRunner(parent string) *Runner {
 	}
 	cmdutil.FixDocs("kpt", parent, c)
 	r.Command = c
-	c.Flags().StringVar(&r.FilenamePattern, "pattern", filters.DefaultFilenamePattern,
-		`Pattern to use for writing files.  
-May contain the following formatting verbs
-%n: metadata.name, %s: metadata.namespace, %k: kind
-`)
+	c.Flags().StringVar(&r.strategy, "strategy", string(kptfilev1alpha2.ResourceMerge),
```

Comment:
Why do we need it as part of get command ? A flag should enhance the behavior of current command but this flag is to record an input for future command(update). What does this mean to user `kpt pkg get --strategy=resource-merge`

Reply:
Since the update strategy is now included in the package Kptfile, we do set it in the `Upstream` section of the Kptfile when we first fork the package. This allows users to specify the strategy that should be set (this defaults to `resource-merge`).

Reply:
Yeah, I get that. But does it make sense to have it as a flag for get command ?

Reply:
I think it is useful to give users the option to specify it when the package is forked. So similar to how we currently let users specify the update strategy for dependencies when they are added to the `dependencies` section.

Reply:
I am still not convinced if this flag should be a part of `get` command. But if you feel it is useful then I am fine with it.